### ISSUE TITLE
Memory leak on rotation

### ISSUE THREAD

**Issue by fvilarino** [futuresimple/android-floating-action-button#177]
I have added this library to my app. I notice that, whenever I rotate the app (destroy/create), the memory used increases and is not freed. I have used the memory tracker, and it points to the Drawable in createCircleDrawable function - every time I rotate, this is called and there are a bunch of byte[] allocated on the heap, which are apparently not freed.

**Comment by chalup** [comment]
I have used MAT (http://android-developers.blogspot.com/2011/03/memory-analysis-for-android.html) with sample app and I do not see any leaked objects after rotation. Have you triggered the GC manually before fetching the heap dump?

**Comment by fvilarino** [comment]
Yes, I have triggered a GC after rotating, the free memory does not increase. Appears to be very easy to see on Android Studio's memory tracker.

Please see this screen capture: https://www.dropbox.com/s/rj3iplz91ck64r6/Selection_003.png?dl=0
